pages/events.tsx

```diff
@@ -70,33 +70,33 @@ const EventsPage = () => {
       })
   }, [setLoading])
 
+  let futureEvents = events.filter((event) => Date.parse(event.date) > Date.now());
```

Comment:
what happens when parsed-date === Date.now() :-)

Reply:
I agree that the Sanity should return a Date object.. the date input type is a string by default (from [docs](https://www.sanity.io/docs/date-type)) and I'm looking for a workaround (it's been hard)